src/library/scala/collection/GenTraversableOnce.scala

```diff
@@ -80,6 +94,9 @@ trait GenTraversableOnce[+A] extends Any {
   def size: Int
 
   /** Tests whether the $coll is empty.
+    *
+    * Note: Implementations in subclasses that are not repeatedly traversable must take
```

Comment:
I phrased it like this because the consumer side is already documented in the overridden method in Iterator, the only non-Traversable subclass of TraversableOnce. You only really see the version of the doc in TraversableOnce when you want to subclass it directly.